install/playground/contour-http-proxy.yaml

```
@@ -0,0 +1,35 @@
+apiVersion: projectcontour.io/v1
+kind: HTTPProxy
+metadata:
+  name: meshery
+  namespace: meshery
```

```suggestion
  namespace: meshery
    annotations:
      metallb.universe.tf/address-pool: my-ip-space
```

To ensure that only Contour receives the IP assignment.... ?